docs/ad/index.md

```diff
@@ -40,36 +41,37 @@ A feature is the field in your index that you want to find anomalies for. A dete
 {: .note }
 
-1. Select the **Enable feature** check box.
-2. Add the **Feature name**.
-3. For **Feature type**, choose **Field aggregation**, and then choose the aggregation method. Or choose **Custom expression**, and add in your own JSON aggregation query.
-4. Choose a **Field**.
-5. Choose **Save**.
+1. On the **Features** page, select **Add features**.
+1. Enter the **Name** of the feature.
+1. For **Find anomalies based on**, choose the method to find anomalies. For **Field Value** menu, choose the field and the **aggregation method**. Or choose **Custom expression**, and add in your own JSON aggregation query.
+1. Preview sample anomalies and adjust the feature settings if needed.
+1. Choose **Save**.
+1. Choose between automatically starting the detector (recommended) or manually starting the detector at a later time.
 
 ### Step 3: Observe the results
 
-The **Total anomalies** graph plots the anomaly grade with the corresponding measure of confidence.
+The **Live anomalies** chart shows you the live anomaly results for the last 20 minutes.
 
-Anomaly grade is a number between 0 and 1 that indicates the level of severity of how anomalous a data point is. An anomaly grade of 0 represents “not an anomaly,” and a non-zero value represents the relative severity of the anomaly. The confidence score is an estimate of the probability that the reported anomaly grade matches the expected anomaly grade. Confidence increases as the model observes more data and learns the data behavior and trends. Note that confidence is distinct from model accuracy.
+The **Anomaly history** chart plots the anomaly grade with the corresponding measure of confidence.
 
-The **Features** graph plots the features based on the aggregation method. On the top-right corner, you can vary the date-time range of the detector.
+Anomaly grade is a number between 0 and 1 that indicates the level of severity of how anomalous a data point is. An anomaly grade of 0 represents “not an anomaly,” and a non-zero value represents the relative severity of the anomaly. The confidence score is an estimate of the probability that the reported anomaly grade matches the expected anomaly grade. Confidence increases as the model observes more data and learns the data behavior and trends. Note that confidence is distinct from model accuracy.
 
-To see all the anomalies that are detected, choose the **Anomalies** tab.
+The **Feature breakdown** graph plots the features based on the aggregation method. On the top-right corner, you can vary the date-time range of the detector.
 
-### Step 4: Adjust the model
+### Step 4: Set up alerts
 
-You can enable or disable features or fine tune the time interval to minimize any false positives.
+To create a monitor to send you notifications when any anomalies are detected, choose **Set up alerts**. For steps to create a monitor and set notifications based on your anomaly detector, see [Monitor](../alerting/monitors/).
```

Comment:
I feel we can add more details to the part covering “setting up monitor” as the interaction between monitoring and anomaly detection is not easy to understand.    This is my write-up:

Anomaly detection (AD) and Alerting plugins interact using anomaly result index (.opendistro-anomaly-results).  There is a scheduled job (like a cron job) running behind each AD detector and each Alerting monitor.   The scheduled job behind a detector writes to anomaly result index, while the scheduled job behind a monitor reads from the anomaly result index. An interval defines how frequent the determines how frequently the scheduled job runs. How frequently a detector writes to the anomaly result index determines how frequently a monitor reads from the anomaly result index. To a monitor, we need to decide on an appropriate monitor interval based on detector interval.  Otherwise, the alerting monitor may miss reading results. Suppose we have a 5-minute interval AD detector.  If our monitor has the same interval as the detector interval,  the monitor reads the beginning time of the 5-minute bucket every 5 minutes.  So, the AD job currently processing the bucket of data between 11:55 and 12:00, will be indexed any anomalies found in the timeframe with a timestamp a bit later after 12:00.  The clock time at which this anomaly record is available for reading could be as late as around 12:01 due to some delay between when a document containing the anomaly record is written and when the document is available for query (e.g., index refresh time). As such, a monitor triggered at 12:00, which looks back for anomalies with timestamps as early as 11:55, will not see anything because the anomaly record hasn’t even been available for search yet (and won’t be for another one minute).

Timeline:
12:05:00 The detector fetches user data within [12:00:00, 12:05:00] and writes an anomaly result at 12:05:00. The record is available in 12:06:00, 1 minute due to some delay between when a document containing the anomaly record is written, and when the document is available for query.
12:05:00 The monitor reads anomaly results written within [12:00:00, 12:05:00] and does not get any anomaly results since it is available at 12:06:00.
12:10:00 The detector fetches user data within [12:05:00, 12:10:00] and writes an anomaly result at 12:10:00.
12:10:00 The monitor reads anomaly results within [12:05:00, 12:10:00]and does not get any anomaly result since the record is available in 12:11:00)

To solve the issue, we need Alerting monitor’s interval to be at least twice of detector interval to not miss any AD results.  Consider that our monitor interval is 10 minutes.

Timeline:
12:05:00 The detector fetches user data within [12:00:00, 12:05:00] and writes an anomaly result at 12:05:00. The record is available for search at 12:06:00.
12:10:00 The monitor reads anomaly results within [12:00:00, 12:10:00] and find anomaly result updated at 12:05:00 since the record is available for search at 12:06:00)
12:10:00 The detector fetches user data within [12:05:00, 12:10:00] and writes an anomaly result at 12:10:00. The record is available for search at 12:11:00.
12:15:00 The detector fetches user data within [12:10:00, 12:15:00] and writes an anomaly result at 12:15:00. The record is available for search at 12:16:00.
12:20:00 The monitor reads anomaly results within [12:10:00, 12:20:00] and gets two anomaly results updated at 12:10:00 and 12:15:00.

When you create a monitor using Kibana, we will generate a default monitor interval that is two times of detector interval. Whenever you update a detector’s interval, you should remember to update an associated monitor interval as well. 

Due to the loose coupling of Alerting and Anomaly detection plugins, we won’t clean up a monitor if we delete/stop a detector.  If you keep your monitor after deleting/stopping a detector, the monitor is running and query for nothing.  You should remember to remove associated monitors if you delete/stop a detector.

Reply:
Added. Please review.